package.json

```diff
@@ -6,7 +6,9 @@
     "node-hc": "bin/appmetrics-cli.js"
   },
   "dependencies": {
+    "heapdump": "0.x",
     "nan": "2.x",
+    "nodereport": "1.x",
```

Comment:
this would have to be moved to `optionalDependencies`, because it doesn't build on 0.12 which is supported by appmetrics.

Also, both heapdump and nodereport should be listed below in bundleDependencies, so they get bundled.

And also, I belive that appmetrics actually uses/generates a slightly different package.json when building the appmetrics package for release, one that doesn't depend on nan, for example, so @sjanuary, that one will need updating to reflect the bundle spec changes.